AfricanReligions/localisation/afr_general_localizations.csv

```diff
@@ -213,6 +213,8 @@ vanilla_pick_main_deity;Pick Patron Deity;;;;;;;;;;;;x
 vanilla_pick_main_deity_desc;There are many gods and goddesses who rule over different aspects of our life. While all of them should be honored properly, it is common for people to pick a deity whom they pray and sacrifice to more often.;;;;;;;;;;;;x
 bori_pick_main_deity;Pick Patron Bori;;;;;;;;;;;;x
 bori_pick_main_deity_desc;It is good to have a protective spirit, therefore it would be a good idea to choose one.;;;;;;;;;;;;x
+amazigh_pick_main_deity;Pick Patron Deity;;;;;;;;;;;;x
```

Comment:
I don't think we need different localization for this one, so we don't need these localization keys.  Amazigh will just use the regular "not Bori" localization.

Reply:
There's a different decision localisation on the spreadsheet for the decision but I can remove it if you wish.

Reply:
I think it's more important to be consistent here.  I mainly wanted different text for Bori because the bori are not really deities, per se.